xchange-core/src/main/java/org/knowm/xchange/dto/account/DepositAddress.java

```diff
@@ -0,0 +1,50 @@
+package org.knowm.xchange.dto.account;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public final class DepositAddress implements Serializable {
```

Comment:
we already have a very generic interface WithdrawFundsParams which you can extend to add a tag 
we also already have this address in DefaultWithdrawFundsParams which already has the address
we also already have RippleWithdrawFundsParams that adds a tag for ripple and MoneroWithdrawFundsParams with add paymentId <- this is a little unfortunate as they are in essence the same this

please remove DepositAddress , add destinationTag to DefaultWithdrawFundsParams with the following documentation:

```
  /**
   * Various cryptocurrencies allow to add an additional identifier / tag to you transaction which helps them
   * to process you payment. Ripple for example has a tag, Monero has a payment id.
   */
  public String getDestinationTag() {
```
and us it instead in Binance and Coinbase

in another PR I'll then refactor the code to deprecate RippleWithdrawFundsParams and MoneroWithdrawFundsParams and add some better WithdrawFundsParams  interfaces